proto/executor.proto

```diff
@@ -22,6 +22,8 @@ enum ExecType {
 	TypeLimit = 5;
 	TypeStreamAgg = 6;
 	TypeJoin = 7;
+	TypeExchanger = 8;
+	TypeExchangerClient = 9;
```

Comment:
rename to exchangeSender and ExchangeReceiver ?

Reply:
ok